blues/version.py

```diff
@@ -2,6 +2,6 @@
 # This file is automatically generated by setup.py
 short_version = '0.0.1'
 version = '0.0.1.dev2'
-full_version = '0.0.1.dev2-0d9bf4c'
-git_revision = '0d9bf4cc2ca5931ac122c6375ae1af5f73a31df8'
+full_version = '0.0.1.dev2-571a2d7'
+git_revision = '571a2d7dce71d41505e8f9c0176645e6cce53726'
```

Comment:
I adopted John's setup.py installation script from the Yank repo: https://github.com/choderalab/yank/blob/master/setup.py

I've gone ahead and added a credits line too.